examples/basic/android/app/src/main/java/com/videoplayer/MainApplication.java

```diff
@@ -24,8 +24,8 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.asList(
         new MainReactPackage(),
-        new ReactVideoPackage()
-      );
+            new ReactVideoPackage()
+          );
```

Comment:
I think all these changes can be reverted, likely caused by CLI link

Reply:
yes, i used react-native link again

Reply:
So, can we revert these changes to indentation (to preserve 2 spaces tabs) and import order? :)

Reply:
i think it is not necessary :D

Reply:
I agree with @vokhuyetOz here, these changes are made by the cli and will be introduced again next time someone runs the `link`

Reply:
Just FYI, starting from RN 0.60 you likely won't run link ever again :)